common.js

```diff
@@ -1686,6 +1686,445 @@ const allSoloCars = {
       'all': ['stx', 'fsp'],
     },
   },
+  'Datsun': {
+    '2000': {
+      'all': ['es']
+    },
+    '240Z': {
+      'all': ['es', 'str', 'bsp']
+    },
+    '240Z, 260Z, & 280Z (incl. 2+2)': {
```

Comment:
I'd like to avoid models that actually have multiple models within. So let's get rid of this and just add `fp` to the existing `240Z`, `260Z`, and `280Z` models.

Reply:
Thanks! 
Done!